lib/jira/resource/agile.rb

```diff
@@ -31,6 +31,18 @@ def self.get_sprint_issues(client, sprint_id, options = {})
         parse_json(response.body)
       end
 
+      def self.get_projects_full(client, board_id, options = {})
+        response = client.get("/rest/agile/1.0/board/#{board_id}/project/full")
```

Comment:
Right, I wrote these methods on my local copy of the gem before I decided to do a PR I had not paid attention to the changes, I'll fix that.